api/tools/src/main/java/org/pmiops/workbench/tools/ConfigLoader.java

```diff
@@ -103,6 +99,9 @@ public CommandLineRunner run(ConfigDao configDao) {
   }
 
   public static void main(String[] args) throws Exception {
-    new SpringApplicationBuilder(ConfigLoader.class).web(false).run(args);
+    new SpringApplicationBuilder(CommandLineToolConfig.class)
+        .child(ConfigLoader.class)
```

Comment:
what does the `child` param do?

Reply:
The child config can override beans defined in the parent: https://www.baeldung.com/spring-boot-context-hierarchy

Reply:
Very interesting. Thanks.